yarn-project/aztec-cli/src/index.ts

```diff
@@ -1,21 +1,71 @@
 #!/usr/bin/env node
 import { createLogger } from '@aztec/foundation/log';
 import { Command } from 'commander';
+import { mnemonicToAccount, privateKeyToAccount } from 'viem/accounts';
+import { createDebugLogger } from '@aztec/foundation/log';
+import { createAztecChain, deployL1Contracts } from '@aztec/blockchain';
+import { deployL2Contract } from './deploy_l2_contract.js';
+
+const logger = createDebugLogger('aztec:cli');
 
 const program = new Command();
 const log = createLogger('aztec:aztec-cli');
 
+/**
+ * Function to execute the 'deployRollupContracts' command.
+ * @param rpcUrl - The RPC URL of the ethereum node.
+ * @param apiKey - The api key of the ethereum node endpoint (leave as '' if no api key is used).
+ * @param privateKey - The private key to be used in contract deployment.
+ * @param mnemonic - The mnemonic to be used in contract deployment.
+ */
+async function deployRollupContracts(rpcUrl: string, apiKey: string, privateKey: string, mnemonic: string) {
+  const account = !privateKey ? mnemonicToAccount(mnemonic!) : privateKeyToAccount(`0x${privateKey}`);
+  const chain = createAztecChain(rpcUrl, apiKey);
+  await deployL1Contracts(chain.rpcUrl, account, chain.chainInfo, logger);
+}
+
 /**
  * A placeholder for the Aztec-cli.
  */
 async function main() {
   program
     .command('run')
-    .argument('<cmd>', 'command')
+    .argument('<cmd>', 'Command')
     .action((cmd: string) => {
       log(`Running '${cmd}'...`);
     });
 
+  program
+    .command('deployRollupContracts')
+    .argument(
+      '[rpcUrl]',
+      'Url of the ethereum host. Chain identifiers localhost and testnet can be used',
+      'http://localhost:8545',
+    )
+    .argument('[apiKey]', 'Api key for the ethereum host', '')
```

Comment:
default value here too, unless undefined is only relevant for sequencer in which case ignore this comment